Description: Fix several hypen-used-as-minus-signs issues in tsmarty2c.1
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: https://github.com/smarty-gettext/smarty-gettext/issues/2 
--- a/tsmarty2c.1
+++ b/tsmarty2c.1
@@ -12,9 +12,9 @@
 'nh
 .fi
 .ad l
-\fBtsmarty2c\fR <templ-file1-or-dir1> \\
-          [<templ-file2-or-dir2>] \\
-          [... <templ-fileN-or-dirN>]
+\fBtsmarty2c\fR <templ\-file1\-or\-dir1> \\
+          [<templ\-file2\-or\-dir2>] \\
+          [... <templ\-fileN\-or\-dirN>]
 
 .SH DESCRIPTION
 .PP
@@ -29,7 +29,7 @@
 The output also contains comments that point to the location of the file 
 that the string has been ripped from.
 .PP
-You can use the '--add-comments' parameter of xgettext to make it include
+You can use the '\-\-add-comments' parameter of xgettext to make it include
 these comments inside the PO file (suggested by Mike van Lammeren).
 .SH SEE ALSO
 xgettext(1)
